### PR TITLE
Update target framework to .Net 6 for tests project

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <IsPackable>false</IsPackable>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
-    <TargetFrameworks>net5.0;net48</TargetFrameworks>
+    <TargetFrameworks>net6.0;net48</TargetFrameworks>
```
Comment (pavel-mikula-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj:7]: `scripts\build\build-utils.ps1` needs to be updated a little as well.

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
There one more hidden file to update

File: scripts/build/build-utils.ps1 [hunk 2]
```diff
@@ -145,18 +145,18 @@ function Invoke-UnitTests([string]$binPath, [string]$buildConfiguration) {
         }
     $testDirs = $testDirs | Select-Object -Uniq
 
-    Write-Header "Running unit tests .NET Framework 4.8"
-    & (Get-VsTestPath) $testFiles /Logger:"console;verbosity=minimal"  /Parallel /Enablecodecoverage /InIsolation  /TestAdapterPath:$testDirs
-    Test-ExitCode "ERROR: Unit Tests execution FAILED."
+#     Write-Header "Running unit tests .NET Framework 4.8"
+#     & (Get-VsTestPath) $testFiles /Logger:"console;verbosity=minimal" /Parallel /Enablecodecoverage /InIsolation  /TestAdapterPath:$testDirs
+#     Test-ExitCode "ERROR: Unit Tests execution FAILED."
```
Comment (pavel-mikula-sonarsource) [inline scripts/build/build-utils.ps1:150]: Debug artifact? Doesn't sem intentional
```suggestion
     Write-Header "Running unit tests .NET Framework 4.8"
     & (Get-VsTestPath) $testFiles /Logger:"console;verbosity=minimal" /Parallel /Enablecodecoverage /InIsolation  /TestAdapterPath:$testDirs
     Test-ExitCode "ERROR: Unit Tests execution FAILED."
```

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
One more glitch

File: scripts/build/build-utils.ps1 (hunk 2 above)
Comment (costin-zaharia-sonarsource) [reply scripts/build/build-utils.ps1:150]: oh yes. I was testing and forgot to remove the comments. Thaks

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**